app/Livewire/Pages/Forum/DetailThread.php

```diff
@@ -56,7 +56,7 @@ public function deleteAction(): Action
             ->action(function (): void {
                 $this->thread->delete();
 
-                undoPoint(new ThreadDeleted($this->thread));
+                undoPoint(new ThreadCreated($this->thread));
```

Comment:
Tu supprimes le model et tu passe le model comment ? A la ligne 59 `$this->thread` peut etre null parce que tu l'as supprimé juste avant. Tu dois mettre ça dans une transaction et tu supprimes le point avant de supprimer le model pas l'inverse